go.mod

```diff
@@ -29,7 +29,7 @@ replace github.com/metal3-io/cluster-api-provider-metal3/api => ./api
 
 replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.4.2
 
-replace github.com/docker/distribution => github.com/docker/distribution v2.8.1+incompatible
+replace github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible
```

Comment:
Maybe a stupid question, but why are we needing these `replace` lines instead of just `require`?

Reply:
Sometimes for pinning versions replace is required especially if dependancy is required indirectly